packages/storefront-lwc/src/modules/commerce/data/basket.js

```diff
@@ -105,14 +105,64 @@ class Basket {
             });
     }
 
-    // TODO : wire this call with BFF
-    removeFromBasket(index) {
-        let basket = this.getCurrentBasket();
-        if (index > -1) {
-            basket.splice(index, 1);
-        }
-
-        this.updateBasket(basket);
+    removeItemFromBasket(itemId) {
+        return apiClient
+            .mutate({
+                mutation: gql`
+                    mutation {
+                        removeItemFromBasket(itemId: "${itemId}") {
+                            basketId
```

Comment:
We could probably put all of these attributes in a string at the top of the file, and use it both here and in `getCurrentBasket`

Reply:
Good point .. Done!